Jenkinsfile

```diff
@@ -4,6 +4,7 @@
 
 // mxnet libraries
 mx_lib = 'lib/libmxnet.so, lib/libmxnet.a, dmlc-core/libdmlc.a, nnvm/lib/libnnvm.a'
+mx_mkldnn_lib = 'lib/libmxnet.so, lib/libmxnet.a, lib/libiomp5.so, lib/libmklml_gnu.so, lib/libmkldnn.so, lib/libmkldnn.so.0, lib/libmklml_intel.so, dmlc-core/libdmlc.a, nnvm/lib/libnnvm.a'
```

Comment:
What's libiomp5? What's libmklml_gnu used for if this is about mkldnn?

Reply:
libiomp5 is the "commercial build" of Intel's OMP library
Also, I see libmklml_gnu and libmklml_intel.  I'm pretty sure these are the same libraries compiled with different compilers and should not be used at the same time (if at all with mkldnn)

Reply:
These libraries are provided by Intel people. They link the library in such a way. But you are right. Maybe we shouldn't link both Intel's OMP library and GNU OMP library.